lib/plugins/aws/invoke-local/index.js

```diff
@@ -253,7 +253,7 @@ class AwsInvokeLocal {
       );
     }
 
-    if (['python2.7', 'python3.6', 'python3.7', 'python3.8', 'python3.9'].includes(runtime)) {
+    if (['python2.7', 'python3.6', 'python3.7', 'python3.8', 'python3.9', 'python3.10'].includes(runtime)) {
```

Comment:
Prettier is failing in the CI, this should make it work:
```suggestion
    if (
      ['python2.7', 'python3.6', 'python3.7', 'python3.8', 'python3.9', 'python3.10'].includes(
        runtime
      )
    ) {
```

Reply:
@giuliano-oliveira Thank you for the suggestion ! I should have followed the documentation below.
https://github.com/serverless/serverless/blob/main/CONTRIBUTING.md#verifying-prettier-formatting